src/main/java/net/shadowmage/ancientwarfare/npc/init/AWNPCSounds.java

```diff
@@ -7,11 +7,130 @@
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import net.minecraftforge.fml.common.registry.GameRegistry.ObjectHolder;
 import net.minecraftforge.registries.IForgeRegistry;
+import net.shadowmage.ancientwarfare.core.util.InjectionTools;
 import net.shadowmage.ancientwarfare.npc.AncientWarfareNPC;
 
+import java.util.HashMap;
+import java.util.Map;
+
 @ObjectHolder(AncientWarfareNPC.MOD_ID)
 @Mod.EventBusSubscriber(modid = AncientWarfareNPC.MOD_ID)
 public class AWNPCSounds {
+
+	public static Map<String, SoundEvent> npcSoundEvents = new HashMap<>();
+
+	public static final SoundEvent ENTITY_BARBARIAN_ATTACK = InjectionTools.nullValue();
```

Comment:
I feel like all this ENTITY_ prefixing is just quite a bit of additional noise. Vanilla may do it, but it may be easier on eyes to just remove it. The same applies to the event names in the sounds.json. So unless you feel strongly otherwise let's remove these prefixes.